scripts/linux/electron-installer-appimage.sh

```diff
@@ -105,7 +105,6 @@ fi
 if [ -z "$TMPDIR" ]; then
   TMPDIR=$(mktemp -d)
```

Comment:
...although it _does_ mean that we're back to the situation with mktemp creating randomly-named directories, which I commented on in another issue as being make-unfriendly... :-S